static/app.js

```diff
@@ -262,4 +278,40 @@
 		}
 	})
 
+	app.controller('areaHealthModificationCtrl', function($scope, $modalInstance, characters){
+		$scope.areaDamage = 0;
+
+		$scope.characters = characters;
+		$scope.characters.forEach(function (character) {
+			character.damaged = false;
```

Comment:
You shouldn't add damaged and damageHalved properties to the character prototype.